build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -343,6 +343,18 @@ class SeleniumWebDriverController {
     fs.writeFile(path, imageString, 'base64', function() {});
   }
 
+  /**
+   * Evaluate the given function
+   * @param {function()} fn
+   * @param {...*} unusedArgs
+   * @return {!ControllerPromise}
+   * @override
+   */
+  eval_(fn, unusedArgs) {
```

Comment:
The style guide says trailing `_` for `@private` methods, fields, etc. but does not say to use them for `@package`. Should remove the `_`.